### DISCUSSION TITLE
"maintainers" key isn't being set on package registry

### DISCUSSION THREAD

**Discussion by PatrickStankard** [community/community#129220] [npm]
I noticed that the "maintainers" key isn't being set when publishing a new version of a package to the registry: https://registry.npmjs.org/homebridge-fujitsu-airstage/latest

I did a no-op publish (bumping from 1.0.6 -> 1.0.7) to see if it was a one-off, but it happened on both versions

Here is a previous version that had the "maintainers" key set, as expected: https://registry.npmjs.org/homebridge-fujitsu-airstage/1.0.5

**Comment by NorthernMan54** [comment]
This is a npm supplied documented field.

From - https://docs.npmjs.com/cli/v10/configuring-npm/package-json#people-fields-author-contributors

**`npm also sets a top-level "maintainers" field with your npm user info.`**

**Reply by grandmas-favorite** [reply]
Nice catch @NorthernMan54! 😄

**Comment by dkerr64** [comment]
And https://github.com/npm/www/issues/133#issuecomment-284906561 npm states themselves...

_"The "maintainers" field in the top-level of the registry metadata (ie, not versioned) is the **npm-controlled** list of the npm usernames of the people with permission to write to that package. This can be modified via the npm owners command."_ (emphasis mine).
